jobs/gorouter/templates/gorouter.yml.erb

```diff
@@ -28,11 +28,11 @@ def parse_ip (ip, var_name)
   end
 end
 
-def validate_max_header_bytes (bytes)
-  if bytes < 1 or bytes > 1024 * 1024
-    raise "Invalid router.max_header_bytes value. Must be between 1 and 1,048,576 bytes"
+def validate_max_header_kb (bytes)
```

Comment:
I think the logic in this function is correct, but I think the `bytes` variable should now be called `kb` or `kilobytes`. When that value is multiplied 1024, _then_ you get the number of bytes.